express_relay/sdk/js/src/examples/simpleSearcherLimo.ts

```diff
@@ -112,41 +119,49 @@ class SimpleSearcherLimo {
       order.address,
       bidAmount,
       new anchor.BN(Math.round(Date.now() / 1000 + DAY_IN_SECONDS)),
-      this.chainId
+      this.chainId,
+      this.expressRelayConfig!.relayerSigner,
+      this.expressRelayConfig!.feeReceiverRelayer
     );
 
+    bid.transaction.recentBlockhash = opportunity.blockHash;
+    bid.transaction.sign(this.searcher);
+    return bid;
+  }
+
+  async opportunityHandler(opportunity: Opportunity) {
+    if (!("order" in opportunity))
+      throw new Error("Not a valid SVM opportunity");
```

Comment:
Is it okay to throw when getting invalid opp?

Reply:
If you have subscribed to a SVM chain and receive EVM opportunities, I think you should throw and it's fine here